### DISCUSSION TITLE
Normalize input data

### DISCUSSION THREAD

**Discussion by biggerlambda** [Lightning-AI/pytorch-lightning#6832] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Does pytorch lightning have an easy way to normalize input data? I am feeling lazy today :)

**Comment by aribornstein** [comment]
What kind of data are you dealing with text, images, tabular? The Lightning data module provides hooks, where you can apply, transforms but it will help to better understand what kind of data you want to normalize.

**Reply by biggerlambda** [reply]
It's tabular data. Looking to find some examples.
